### ISSUE TITLE
Enhancement: suppress_tests count or tests_from_junit count

### ISSUE THREAD

**Issue by fishercraigj** [lyndsey-ferguson/fastlane-plugin-test_center#18]
Total extreme NIT as this would be a nice to have feature for people like me who are uber detailed in their reporting!  I'm sure I can figure out how to do a query like this with xcodebuild but I'd like to avoid if possible as things will get messy.

IMHO it would be great if we could somewhere get a tally/list of disabled tests so that I can note that in my output across my CI.  Maybe have it added to the junit file somehow?  Don't know, just thinking out loud.  I'm really just starting to move away from Xcode Server and into Jenkins so junit work isn't my forte as I'm used to the built in craptastic Apple CI.  So forgive me if I'm missing something painfully obvious.  :)

Here's a crude example of what I'm trying to do:

```
# Disable tests
disabled_tests = suppress_tests(
      xcodeproj: 'myproj.xcodeproj',
      tests: ['UnitTests/GlobalStringHelperTests'],
      scheme: 'myproj'
    )

# run tests
    retryCount = 3
    multi_scan(
         devices: ["iPhone 7 (10.3.1)"],
         scheme: 'myproj',
         workspace: File.absolute_path('../myproj.xcworkspace'),
         try_count: retryCount)

# get results
result = tests_from_junit(junit: './fastlane/test_output/report.junit')
    failedCount = result[:failed].count
    passingCount = result[:passing].count
    testCount = failedCount + passingCount 

# send out results to CI (example using UI.message)
    UI.message("<-------- Total tests: #{testCount}") 
    UI.message("<-------- Count of passing tests: #{passingCount}") 
    UI.message("<-------- Count of failed tests: #{failedCount}") 
    UI.message("<-------- List of failed tests: #{result[:failed]}") 
    # NICE TO HAVE: count AND list of disabled_tests that were not run
    # UI.message("<-------- Disabled tests count: ")
    # UI.message("<-------- List of disabled tests: ")
```

**Comment by lyndsey-ferguson** [comment]
Are you just looking to see a nice block of output with this info? If so, I would use `FastlaneCore::PrintTable`. You can see how I use it [here](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L13). If so, that's pretty easy.

Would you be open to creating a PR to add this?

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.0.1](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/tree/v3.0.1)
